updater/site/github_release.py

```diff
@@ -0,0 +1,61 @@
+import requests
+import re
+from bs4 import BeautifulSoup
+
+from updater.site.abstract_site import AbstractSite
+from updater.site.github import GitHub
+from updater.site.enum import GameVersion
+
+
+class GitHubRelease(AbstractSite):
+    _URLS = ['https://www.github.com/', 'https://github.com/']
+
+    session = requests.session()
+
+    def __init__(self, url: str):
+        super().__init__(url, GameVersion.agnostic)
+
+    @classmethod
+    def handles(cls, url: str) -> bool:
+        v = bool(re.match('^https://(www.)?github.com/[^/]+/[^/]+/releases/?$', url))
+        return bool(re.match('^https://(www.)?github.com/[^/]+/[^/]+/releases/?$', url))
```

Comment:
This works great. Only because you asked for a bit of feedback, I think if I were to write this, I'd have just made it something like:
```suggestion
        return any(url.startswith(allowed_url) and url.endswith('/releases') for allowed_url in _URLS)
```